src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -216,22 +216,23 @@ private function getContainerDefinitionData(Definition $definition, $omitTags =
             'synthetic' => $definition->isSynthetic(),
             'lazy' => $definition->isLazy(),
         );
+
         if (method_exists($definition, 'isSynchronized')) {
-            $data['synchronized'] = $definition->isSynchronized();
+            $data['synchronized'] = $definition->isSynchronized(false);
         }
 
         $data['abstract'] = $definition->isAbstract();
         $data['file'] = $definition->getFile();
 
-        if ($definition->getFactoryClass()) {
+        if ($definition->getFactoryClass(false)) {
             $data['factory_class'] = $definition->getFactoryClass();
```

Comment:
I left it as is because I thought that we would need the deprecation notice to be raised if the `factory_class` keyword is used. As long as the framework will now use the `factory` keyword, we shouldn't have any deprecations raised anymore. Only for end users' codebase. Should I silence the deprecation notice then?